src/screens/Login/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { useRef, useState, useEffect } from 'react'
+import Card from '../../components/card/Card'
+import backgroundImage from '../../images/background_image.png'
+import Background from '../../components/background/Background'
+import Header from '../../components/title/header/Header'
+import InputField from '../../components/input/inputfield/InputField'
+import Button from '../../components/button/Button'
+import axiosInstance from '../../utils/api/apiservice'
+// Component handling the login functionality for the user
+const Login = () => {
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth)
+  const imageUrl = windowWidth >= 480 ? backgroundImage : 'none'
+
+  useEffect(() => {
+    const handleWindowResize = () => {
+      setWindowWidth(window.innerWidth)
+    }
+
+    window.addEventListener('resize', handleWindowResize)
+
+    return () => {
+      window.removeEventListener('resize', handleWindowResize)
+    }
+  }, [])
+  const userRef = useRef<HTMLInputElement>(null)
+
+  // TODO: these variables will be used for the API call to backend - to be uncommented then
+  const [user, setUser] = useState('')
+  const [password, setPassword] = useState('')
+
+  const handleLogin = async (event: any) => {
+    // axios api
+    event.preventDefault()
+    try {
+      const response = await axiosInstance.post('/user/login', {
+        email: user,
+        password: password,
+      })
+      console.log(response)
+    } catch (err) {
+      console.log(err)
```

Comment:
How would this error be visible to user on screen ?

Reply:
This error handling is not complete yet. The design for showing error is not present till then added here as console output. Will be removed once design is present.